sirbot_pyslackers/endpoints/slack/messages.py

```diff
@@ -42,14 +49,27 @@ async def stock_quote(message, app):
     if not match:
         return
 
-    asset_class, symbol = match.group("asset_class"), match.group("symbol")
+    asset_class, symbol, currency = (
+        match.group("asset_class"),
+        match.group("symbol"),
+        match.group("currency"),
+    )
+
+    currency = (
+        match.group("currency")
+        if match.group("currency") in FIAT_CURRENCY.keys()
```

Comment:
you can keep this, but just as a comment in case you are interested: `in` in a dict will check the keys: `foo in {'foo': 'bar'}` will be True

Reply:
I can drop the `.keys()` when I update the regex since I'll make another push anyway.